PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchRemoteRunState.kt

```diff
@@ -96,28 +104,13 @@ open class SparkBatchRemoteRunState(private val sparkSubmitModel: SparkSubmitMod
         }
     }
 
-    @Throws(ExecutionException::class)
-    override fun checkSubmissionParameter() {
-        val parameter = getSubmitModel().submissionParameter
-
-        if (parameter.clusterName.isNullOrBlank()) {
-            throw ExecutionException("The ${getSubmitModel().sparkClusterTypeDisplayName} to submit job is not selected, please config it at 'Run/Debug configuration -> Remotely Run in Cluster'.")
-        }
-
-        if (parameter.artifactName.isNullOrBlank() && parameter.localArtifactPath.isNullOrBlank()) {
-            throw ExecutionException("The artifact to submit is not selected, please config it at 'Run/Debug configuration -> Remotely Run in Cluster'.")
-        }
-
-        if (parameter.mainClassName.isNullOrBlank()) {
-            throw ExecutionException("The main class name is empty, please config it at 'Run/Debug configuration -> Remotely Run in Cluster'.")
-        }
-    }
-
     override fun getSubmitModel(): SparkSubmitModel {
         return sparkSubmitModel
     }
 
     open fun onSuccess(executor: Executor) {
-        createAppInsightEvent(executor, mapOf("IsSubmitSucceed" to "true"))
+        val additionalProperties = mapOf("IsSubmitSucceed" to "true")
+        createAppInsightEvent(executor, additionalProperties)
```

Comment:
With a property `IsDisconnected`?

Reply:
Will fix this in another PR

---
In reply to: [297972702](https://github.com/microsoft/azure-tools-for-java/pull/3201#discussion_r297972702) [](ancestors = 297972702)